app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -1,7 +1,42 @@
 module Sufia
   class Admin::AdminSetsController < ApplicationController
+    include CurationConcerns::CollectionsControllerBehavior
     load_and_authorize_resource
     layout 'admin'
+    self.presenter_class = Sufia::AdminSetPresenter
+
+    def show
+      add_breadcrumb t(:'sufia.controls.home'), root_path
+      add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
+      add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
+      add_breadcrumb 'View Set', request.path
+      super
+    end
+
+    # Override the default prefixes so that we use the collection partals.
+    def _prefixes
```

Comment:
`prepend_view_path` does not have the same effect as `_prefixes`. it's called here: https://github.com/rails/rails/blob/810dff7c9fa9b2a38eb1560ce0378d760529ee6b/actionpack/lib/action_controller/metal/implicit_render.rb#L34

Perhaps we could override `local_prefixes` instead:
https://github.com/rails/rails/blob/3fc0bbf008f0e935ab56559f119c9ea8250bfddd/actionview/lib/action_view/view_paths.rb#L27